src/content/structured/components/index.mdx

```diff
@@ -1,23 +1,15 @@
 ---
 path: "/components"
 
-date: "2023-02-03"
+date: "2023-11-07"
 
 title: "Components"
 
-subTitle: "This section introduces our reusable UI Kit components."
+subTitle: "Components are reusable user interface elements. These are the basic building blocks of designs create consistent, accessible and usable experiences across our missions, departments and partners."
 
 contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/components/index.mdx"
 ---
 
-## Introduction
+<ComponentGallery />
 
-Components are reusable user interface elements. These are the basic building blocks of designs and are used in all digital apps and services.
-
-Our components use our [style tokens](/styles). They make it easier for people to use your app because they can easily recognise how things work.
-
-All components have been tested to meet or exceed [accessibility requirements](/accessibility/requirement).
-
-## Contribute
-
-We are expanding the Design System and UI Kit. You can check out the [roadmap](/community/roadmap) for what's coming. If you can help you can [read more on how to contribute](/community/contribute).
+Can’t find what you’re looking for? We are expanding the Design System and UI Kit. You can check out the [roadmap](/community/roadmap) for what's coming. If you can help add to the system you can [read more on how to contribute](/community/contribute).
```

Comment:
```suggestion
Can’t find what you’re looking for? We are expanding the Design System and UI Kit. You can check out the [roadmap](/community/roadmap) for what's coming. 

Why not help add to the design system? Here's some more information on [how to contribute](/community/contribute).
```

Reply:
I'd be happy with this change if @DBD324 is? :)